metron-stellar/stellar-common/src/main/java/org/apache/metron/stellar/dsl/StellarFunction.java

```diff
@@ -23,4 +23,5 @@
   Object apply(List<Object> args, Context context) throws ParseException;
   void initialize(Context context);
   boolean isInitialized();
```

Comment:
Why not give this a default implementation so that you don't have to add do nothings to the implementations that are out there?

Reply:
Yeah, was adding that as I posted.